index.js

```diff
@@ -27,6 +27,16 @@ module.exports = function (opts) {
 		throw err;
 	}
 
+	// run autofix over as many errors as possible
+	if (opts.fix) {
+		overrides.maxErrors = Infinity;
+	}
+
+	checker.getConfiguration().override(overrides);
```

Comment:
Why do you need this? You're passing the override config to `checker.configure` below anyways.